usaspending_api/references/migrations/0067_create_officers.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-04-13 17:41
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def update_legal_entities(apps, schema_editor):
```

Comment:
Yeah, this is to ensure that there is _always_ an Officer for each legal entity item. It has two bonuses, both of which you point out: the `LegalEntity.officers` pattern always works, and the front end has an easier life of just checking if an officer is null vs. the entire nested object being null.